userspace/libsinsp/sinsp_filtercheck_thread.cpp

```diff
@@ -761,21 +825,104 @@ uint8_t* sinsp_filter_check_thread::extract(sinsp_evt *evt, OUT uint32_t* len, b
 	case TYPE_ENV:
 		{
 			m_tstr.clear();
-
 			uint32_t j;
 			const auto& env = tinfo->get_env();
```

Comment:
as a future improvement get_env() could return a map instead of a vector, in this way it would be easier to extract it